meteor/server/api/playout/playout.ts

```diff
@@ -1050,6 +1058,148 @@ export namespace ServerPlayoutAPI {
 			}
 		)
 	}
+
+	/**
+	 * Triggered from Playout-gateway when TSR has some concrete values to use instead of 'now'
+	 */
+	export async function timelineTriggerTimeForStudioId(
+		studioId: StudioId,
+		results: PeripheralDeviceAPI.TimelineTriggerTimeResult
+	): Promise<void> {
+		await runStudioOperationWithCache(
+			'timelineTriggerTime',
+			studioId,
+			StudioLockFunctionPriority.CALLBACK_PLAYOUT,
+			async (studioCache) => {
+				const activePlaylists = studioCache.getActiveRundownPlaylists()
+
+				if (activePlaylists.length === 1) {
+					const activePlaylist = activePlaylists[0]
+					const playlistId = activePlaylist._id
+					await runPlayoutOperationWithLockFromStudioOperation(
+						'timelineTriggerTime',
+						studioCache,
+						activePlaylist,
+						PlayoutLockFunctionPriority.CALLBACK_PLAYOUT,
+						async () => {
+							const rundownIDs = Rundowns.find({ playlistId }).map((r) => r._id)
+
+							// We only need the PieceInstances, so load just them
+							const pieceInstanceCache = await DbCacheWriteCollection.createFromDatabase(PieceInstances, {
+								rundownId: { $in: rundownIDs },
+							})
+
+							// Take ownership of the playlist in the db, so that we can mutate the timeline and piece instances
+							timelineTriggerTimeInner(studioCache, results, pieceInstanceCache, activePlaylist)
+
+							await pieceInstanceCache.updateDatabaseWithData()
+						}
+					)
+				} else {
+					timelineTriggerTimeInner(studioCache, results, undefined, undefined)
+				}
+			}
+		)
+	}
+
+	function timelineTriggerTimeInner(
+		cache: CacheForStudio,
+		results: PeripheralDeviceAPI.TimelineTriggerTimeResult,
+		pieceInstanceCache: DbCacheWriteCollection<PieceInstance, PieceInstance> | undefined,
+		activePlaylist: RundownPlaylist | undefined
+	) {
+		let lastTakeTime: number | undefined
+
+		// ------------------------------
+		const timeline = cache.Timeline.findOne(cache.Studio.doc._id)
+		if (timeline) {
+			const timelineObjs = deserializeTimelineBlob(timeline.timelineBlob)
+			let tlChanged = false
+
+			_.each(results, (o) => {
+				check(o.id, String)
+
+				logger.info(`Timeline: Setting time: "${o.id}": ${o.time}`)
```

Comment:
should probably be a `logger.debug` or `logger.verbose`.

The same applies to the orther `logger.into` further down

Reply:
This method was moved from another file, so that isnt a new line. But Ive changed to logger.debug